### DISCUSSION TITLE
API - Disconnect / Deactivate current streaming DJ

### DISCUSSION THREAD

**Discussion by BornKillaz** [AzuraCast/AzuraCast#7261] [Q&A] [answered]
Hi.

Is there an API call that disconnects the current live streaming DJ?

I've tried this, but it looks like has no effect, or maybe it's not the "disconnect_deactivate_streamer" parameter that deals with this...

Endpoint: /admin/station/1

Data sent with PUT:  ['disconnect_deactivate_streamer' => 1]

**Comment by BornKillaz** [comment]
Is it possible to add this command to the API?

**Reply by BusterNeece** [reply, marked as answer]
Please be patient. If you look at the volume of support requests we're getting, you'll realize it's really really high right now. More than I can begin to handle as one person. I'm way behind, so you're just going to have to wait. No need to follow up on things, no matter how politely.

To answer your question, such an API endpoint actually exists already: `POST /api/station/(id)/backend/disconnect`.

If you see functionality in our web UI, there's a 100% chance it's doable via the API as well. Just open your browser console (F12), hit the "Network" tab, and look at the URL it's hitting when you do it in the web UI, and that's the API URL you can use yourself.

**Reply by BornKillaz** [reply]
Amazing Buster, thank you very much.

**Comment by BornKillaz** [comment]
Just tested /api/station/{ID}/backend/disconnect and works like a charm. Thank you.

Just a side note: the API call to api/admin/station/{ID} only accepts the numeric station ID, does not accept its string ID. 

I use that one to retrieve the time that the DJ will be disconnected (disconnect_deactivate_streamer).
